### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Fiannn**
- [DEMO LINK](https://Fiannn.github.io/layout_calendar/)
- [TEST REPORT LINK](https://Fiannn.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

**Review by vkryvytskyi — CHANGES_REQUESTED** [review]
image

https://user-images.githubusercontent.com/38065883/120306975-2176eb00-c2db-11eb-83a8-e5e1fcc175b3.mov

Please check that you set correct branch in repo settings

**Comment by Fiannn** [comment]
image
i didnt change anything...

**Comment by Fiannn** [comment]
Анимация

**Comment by Fiannn** [comment]
added proof in gif format

File: src/styles/calendar.scss [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+  @for $nth-day from 1 to 32 {
+    &__day:nth-child(#{$nth-day})::before {
+      content: "#{$nth-day}";
+    }
+  }
+
+  //setting witch day gonna be first in the month
+  
+  $day-week:
```
Comment (vkryvytskyi) [inline src/styles/calendar.scss:53]: image
I can't see this modifier

**Review by vkryvytskyi — CHANGES_REQUESTED** [review]

**Review by vkryvytskyi — APPROVED** [review]
Add class to the calendar and set it to 29 days, to show it works

**Pull request closed without merging**